app/lib/screens/ChatProfileScreen/ChatProfile.dart

```diff
@@ -0,0 +1,1178 @@
+                              ),
+                              border: InputBorder.none,
+                            ),
+                          ),
+                        ),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.symmetric(
+                          horizontal: 16.0,
+                          vertical: 8.0,
+                        ),
+                        child: Row(
+                          children: [
+                            Text(
+                              'Your invite link expires in 24 hours.',
+                              style: TextStyle(
+                                color: Colors.white.withOpacity(0.5),
+                                fontSize: 14.0,
+                              ),
+                            ),
+                            SizedBox(
+                              width: 5,
+                            ),
+                            GestureDetector(
+                              onTap: () {
+                                Navigator.push(
+                                  context,
+                                  MaterialPageRoute(
+                                    builder: (context) =>
+                                        RoomLinkSettingsScreen(
+                                      room: widget.room,
+                                    ),
+                                  ),
+                                );
+                              },
+                              child: Text(
+                                'Edit invite link',
+                                style: TextStyle(
+                                  color: AppCommonTheme.primaryColor,
+                                  fontSize: 14.0,
+                                ),
+                              ),
+                            ),
+                          ],
+                        ),
+                      ),
+                      Expanded(
+                        child: ListView.builder(
+                          controller: scrollController,
+                          itemCount: 10,
+                          itemBuilder: (context, index) {
+                            return Padding(
+                              padding: const EdgeInsets.all(12.0),
+                              child: InviteListView(
+                                isAdded: false,
+                                name: 'Abhishek',
+                              ),
+                            );
+                          },
+                        ),
+                      ),
+                    ],
+                  ),
+                );
+              },
+            );
+          },
+        );
+      },
+    );
+  }
+
+  void handleClick(int item) {
+    switch (item) {
+      case 0:
+        showReportBottomSheet();
+        break;
+    }
+  }
+
+  void showReportBottomSheet() {
```

Comment:
How can I access this modal? I cannot see a 'Report' Button. Or is it not implemented yet?

Reply:
The report button is implemented but only works when the user is not admin as you can see in screenshots above. I made these as per the design provided by David. So if you wanna access the report modal... you will need to pass isAdmin as false in ChatScreen.dart where we navigating to ChartProfile.dart

Reply:
Okay I see.